### DISCUSSION TITLE
Hotger

### DISCUSSION THREAD

**Discussion by Inna333666** [SinyavtsevIlya/NanoECS#3] [General]
Илья, привет, меня зовут Инна Остапенко и я HR менеджер компании Hotger. Мы делаем игры и галопирующими темпами развиваемся. Как ты понимаешь, для разработки игр мне очень нужны дельные разработчики. Мои технические спецы посмотрели твой код и оценили его качество очень положительно, а от них такое не каждый услышит:)
К сожалению, кроме как таким образом больше никак с тобой связаться не могу.
В электронной визитке все мои контакты, я буду просто счастлива, если ты мне напишешь на почту или в телеграмм.
Спасибо тебе заранее,
Инна Остапенко
HR Менеджер
Hotger Corporation
тел: +7 918 522 5310
telegram: @Ostapenko_Inna
e-mail: inna.ostapenko@hotger.com
сайт: www.hotger.com

**Comment by desendo** [comment]
Здравствуйте Инна!расскажите пожалуйста больше про вакансию 24.03.2021, 14:32, "Inna" ***@***.***>: Илья, привет, меня зовут Инна Остапенко и я HR менеджер компании Hotger. Мы делаем игры и галопирующими темпами развиваемся. Как ты понимаешь, для разработки игр мне очень нужны дельные разработчики. Мои технические спецы посмотрели твой код и оценили его качество очень положительно, а от них такое не каждый услышит:)К сожалению, кроме как таким образом больше никак с тобой связаться не могу.В электронной визитке все мои контакты, я буду просто счастлива, если ты мне напишешь на почту или в телеграмм.Спасибо тебе заранее,Инна ОстапенкоHR МенеджерHotger Corporationтел: +7 918 522 5310telegram: @Ostapenko_Innae-mail: ***@***.***сайт: www.hotger.com—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by Inna333666** [comment]
Илья, привет,
рада, что прочел мое сообщение и не обиделся, с удовольствием расскажу и
про вакансию и про историю ее возникновения.
В данном формате наверное смогу только прислать сухое описание.
А вот подробности и все профиты, что сулит сотрудничество с нами давай
обсудим в голосовом формате.
Я могу запланировать конференцию в zoom, скажи только на какое время.
Могу поговорить через дискорд (Инна#4504), а могу чере телеграмм
@Ostapenko_Inna.
Как тебе удобно.
Вакансия:
Middle Unity разработчик

з/п не указана
Hotger <https://rostov.hh.ru/employer/613932>
<https://feedback.hh.ru/article/details/id/5951>
Ростов-на-Дону, Буденновский, 3
<https://rostov.hh.ru/search/vacancy/?isMap=True&vacancy_id=43002641>
<https://rostov.hh.ru/employer/613932>

Требуемый опыт работы: не требуется

Полная занятость, полный день

Компания Hotger больше 14 лет занимается созданием и продвижением
собственных технологических продуктов в области Web, Mobile, Desktop и AR
решений на рынки России, Европы и США.

В активном поиске Middle Unity Разработчика для усиления команды по
разработке игры.

*Задачи, которые предстоит решать:*

   - Разработка мобильных игр на движке Unity для iOS и Android.
   - Активное участие в проектировании архитектуры и реализации игровых фич.
   - Принятие совместных решений с художниками, геймдизайнерами.

*Что мы ждем:*

   - Отличное знание ООП.
   - Знание алгоритмов и паттернов программирования.
   - Уверенное владение C#.
   - Навыки работы в Unity 3D.
   - Знание школьного курса линейной (векторной) алгебры.
   - Большая любовь к играм.
   - Здоровый перфекционизм.
   - Чувство стиля и сочетаемости элементов.
   - Развитое чувство ответственности за свой код.

*Мы предлагаем*:

   - Работу в стабильной, успешной и дружной компании
   - График полного рабочего дня: 5/2, с 10:00 до 19:00
   - Достойную заработную плату (оклад определяется индивидуально по
   результатам собеседования + бонусы)
   - Возможность развития и профессионального роста, участие в
   инновационных проектах
   - Возможность получить опыт работы с высоконагруженными проектами
   - Работу в современном open-space офисе в центре Ростова-на-Дону
   - Оформление по ТК РФ
   - Обучение новейшим технологиям за счёт компании
   - Бесплатные занятия английским 2 раза в неделю, которые оплачивает
   компания
   - Доплата от компании за абонемент в спорт зал в размере 30%
   - Обеды для сотрудников
   - Чай, кофе, печенье.

С уважением,
Инна Остапенко
*HR Менеджер*
*Hotger Corporation*
*тел: +7 918 522 5310*
*e-mail: ***@***.*** ***@***.***>*
*сайт: www.hotger.com <http://www.hotger.com>*




чт, 25 мар. 2021 г. в 11:14, Alexey ***@***.***>:
